packages/material-ui/src/ImageList/ImageList.js

```diff
@@ -104,12 +104,12 @@ ImageList.propTypes = {
    */
   component: PropTypes.elementType,
   /**
-   * The gap between items in px.
+   * The gap between items in pixels.
```

Comment:
Aren't there different kind of pixels e.g. hardware pixels and software pixels? I think the original text wanted to emphasize that these are software i.e. CSS pixels hence the `px` which is used in CSS.

Reply:
An interesting blog post that covers the terminology http://www.gregoryschmidt.ca/writing/when-is-a-pixel-not-a-pixel-pixels-points-dps. `px` could help clarify potential confusion, by making a stronger reference to web pixels. 

There are a couple of other places where we can do the change. Initially, I have proposed pixels, to match, for instance:

https://github.com/mui-org/material-ui/blob/27fe6a30e5e5432036e8a0ce8a409b75dfb7f2dc/packages/material-ui/src/SwipeableDrawer/SwipeableDrawer.d.ts#L58